platforms/core-runtime/java-language-extensions/src/main/java/org/gradle/api/JavaVersion.java

```diff
@@ -322,11 +334,6 @@ public String toString() {
         return versionName;
     }
 
-    // We have to keep this for a while: https://github.com/gradle/gradle/issues/4856
-    private String getName() {
-        return versionName;
-    }
-
```

Comment:
According to this comment, the problem has been fixed 6 years ago in IntelliJ:

https://github.com/gradle/gradle/pull/4857#pullrequestreview-600473167